React/app.json

```diff
@@ -21,7 +21,11 @@
       "**/*"
     ],
     "ios": {
-      "supportsTablet": true
-    }
+      "supportsTablet": true,
+	  "bundleIdentifier": "asegroup1.phone.compass"
+    },
+	"android": {
+		"package": "asegroup1.phone.compass"
```

Comment:
Should IOS and Android have the same package names? Also If you rename the App name also rename the package